### ISSUE TITLE
Writing an empty string stores a 0

### ISSUE THREAD

**Issue by r3mi** [typicode/json-server#43]
Writing an empty string stores a 0
```
> curl -H "Content-Type: application/json" -X POST http://localhost:3000/posts -d'{"id":6,"comment":""}'
{"id":6,"comment":0}

> curl -H "Content-Type: application/json" -X GET  http://localhost:3000/posts/6
{"id":6,"comment":0}
```

**Comment by typicode** [comment]
Hi thanks for the report, published the fix in v0.5.8

**Comment by r3mi** [comment]
thanks, that was quick !

I just realized that there is the same issue for a string with spaces " ", even in 0.5.8
```
curl -H "Content-Type: application/json" -X POST http://localhost:3000/posts -d'{"id":6,"comment":" "}'
{"id":6,"comment":0}
```
and also for various white spaces
```
curl -H "Content-Type: application/json" -X POST http://localhost:3000/posts -d'{"id":6,"comment":"\t\n\r"}'
{"id":6,"comment":0}
```
JavaScript niceties ...

**Comment by typicode** [comment]
Yeah as you say JS niceties :sweat_smile: I'll trim the string

**Comment by typicode** [comment]
Ok, I've added some tests here https://github.com/typicode/json-server/blob/master/test/utils.js#L34 and published v0.5.9.

**Comment by r3mi** [comment]
Works fine.
Thanks again for this very useful tool !
